processor/tailsamplingprocessor/config.go

```diff
@@ -139,6 +139,8 @@ type NumericAttributeCfg struct {
 	MinValue int64 `mapstructure:"min_value"`
 	// MaxValue is the maximum value of the attribute to be considered a match.
 	MaxValue int64 `mapstructure:"max_value"`
+	// If invertMatch is true it means that we will do the vice versa decision.
```

Comment:
I would prefer something closer to what the String's InvertMatch states:

```
	// InvertMatch indicates that values must not match against attribute values.
	// If InvertMatch is true and Values is equal to '123', all other values will be sampled except '123'.
	// Also, if the specified Key does not match any resource or span attributes, data will be sampled.
```

Reply:
It might be worth clarifying that the values are exclusive: a config with 10 as lower and 100 as higher limits means that values at 10 and 100 will NOT be sampled.

Reply:
I totally agree that explanation here should be the same as for string. Fixed. 
Could you please also review my proposal on [GH discussions](https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/24571)